docs/developing/README.md

```diff
@@ -52,11 +56,35 @@ customElements.define('my-component', MyComponent);
 }
 ```
 
-5. Start your app:
+6. Start your app:
 ```bash
 npm start
 ```
 
+## Separate `class` and Definition Files
+We recommend separating your element's `class` declaration from the call to `customElements.define`. Doing so will allow users of your element to easily extend it without polluting the global custom elements registry, for example:
+
+```javascript
+import { html } from 'lit-element';
+import { MyComponent } from './MyComponent.js';
+
+customElements.define('enhanced-component', class extends MyComponent {
+  render() {
+    static get properties() {
+      return {
+        alert: { type: String }
+      }
+    }
+
+    return html`
+      ${super.render()}
+      <style>p { color: red; }</style>
+      <p>${alert}</p>
+    `
+  }
+})
+```
+
```

Comment:
```suggestion
Note: this only makes sense if you expect your web component to be used by others. If you have very app specific components it's probably easier to skip this separation.
Note: a specific usecase would be for example an app which uses an auto registration of custom elements like [careHtml](https://www.npmjs.com/package/carehtml) then they will always be only interested in the class.

```

Reply:
I would reverse the suggestion personally. So for example say, 'if you are writing a public component which will be used by others, it is a good idea to split the definition and the class'.